package/MDAnalysis/topology/ITPParser.py

```diff
@@ -594,7 +594,15 @@ def parse(self, include_dir='/usr/local/gromacs/share/gromacs/top/',
         if all(e.capitalize() in SYMB2Z for e in self.elements):
             attrs.append(Elements(np.array(self.elements,
                          dtype=object), guessed=True))
-
+            warnings.warn(
+                "Element guessing from types has been added temporarily to "
+                "the ITPParser to preserve the previous behavior of "
+                "guessing any masses of particles that were not defined "
+                "in the ITP file as we transition to the new guessing API. "
```

Comment:
```suggestion
                "The elements attribute has been populated by guessing "
                "elements from atom types. This behaviour has been "
                "temporarily added to the ITPParser as we transition "
                "to the new guessing API. "
```
The explanation is too complex for users - we should simplify the warning.